.github/workflows/issue-pr-triage.yml

```diff
@@ -3,33 +3,48 @@ name: Auto-triage to project boards
 on:
   issues:
     types: [opened, labeled]
-  pull_request:
+  pull_request_target:
     types: [opened, labeled]
 
 env:
-  GITHUB_TOKEN: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  COLUMN_NAME: Needs Triage # column where issues will be created
+  REPOSITORY: ${{ github.repository }}
 
 jobs:
   content_project:
     runs-on: ubuntu-latest
+    env:
+      PROJECT_ID: 11873277
+      PROJECT_NAME: Docs PRs and Issues
     name: Triage to Content Project
     steps:
     - name: Triages NEW issues and NEW pull requests to the Content Project
-      uses: srggrs/assign-one-project-github-action@1.2.0
       if: github.event.action == 'opened'
-      with:
-        project: 'https://github.com/newrelic/docs-website/projects/3'
-        column_name: 'Needs Triage'
+      run: |
+        PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+        HEADER="Accept: application/vnd.github.inertia-preview+json"
+        PR_ID=$(gh api -H "$HEADER" "repos/$REPOSITORY/pulls?state=open" --jq ".[] | select(.number == $PR_NUMBER).id")
```

Comment:
I think this could be simplified to just:

```sh
PR_ID=$(gh api repos/$REPOSITORY/pulls/$PR_NUMBER --jq '.id')
```

[Relevant docs](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request)

Reply:
change made